experimental/oak_functions_with_envoy/client/client.yaml

```diff
@@ -0,0 +1,47 @@
+static_resources:
+  listeners:
+    # Accepts TCP streams and tunnels them over HTTP streams.
+    # https://www.envoyproxy.io/docs/envoy/latest/intro/arch_overview/http/upgrades#tunneling-tcp-over-http
+    - name: client_listener
+      address:
+        # Client proxy address to listen on.
+        socket_address:
+          protocol: TCP
+          address: 0.0.0.0
```

Comment:
Does it really need to listen on all addresses? At least could you make it listen only on 127.0.0.1?